# perl.m4: remove local library directory to pass the QA.
# Created by Yoshiteru KAGEYAMA <yoshiteru.kageyama@toshiba.co.jp>

--- a/config/perl.m4	2021-02-25 11:13:05.742239465 +0900
+++ b/config/perl.m4	2021-02-25 11:13:18.926526489 +0900
@@ -103,6 +103,7 @@
 	pgac_tmp1=`$PERL -MExtUtils::Embed -e ldopts`
 	pgac_tmp2=`$PERL -MConfig -e 'print $Config{ccdlflags}'`
 	perl_embed_ldflags=`echo X"$pgac_tmp1" | sed -e "s/^X//" -e "s%$pgac_tmp2%%" -e ["s/ -arch [-a-zA-Z0-9_]*//g"]`
+	perl_embed_ldflags=`echo $perl_embed_ldflags | sed -e "s:-L/usr/local/lib::g"`
 fi
 AC_SUBST(perl_embed_ldflags)dnl
 if test -z "$perl_embed_ldflags" ; then
